src/main/kotlin/uk/gov/justice/digital/hmpps/makerecalldecisionapi/jpa/entity/RecommendationEntity.kt

```diff
@@ -43,6 +45,11 @@ enum class Status {
   DRAFT, DOCUMENT_CREATED, DELETED
 }
 
+data class CustodyStatusOption(
```

Comment:
Minor point, and don't even know if it's the right way to do it in Kotlin, but every recommendation field that choose from options, will use a structure like this. Could use a generic class?

Reply:
Yeh good point, I'll look at putting that in